src/api_manager/service_control/proto.cc

```diff
@@ -89,7 +73,7 @@ struct SupportedLabel {
   const char* name;
   ::google::api::LabelDescriptor_ValueType value_type;
 
-  enum Kind { USER = 0, SYSTEM = 1 };
+  enum Kind { USER = 0, SYSTEM = 1, USER_BY_CONSUMER = 2 };
```

Comment:
this is system label.  It is used in line 1040.  we could not change "Kind".  

may have to add field 

bool by_consumer_only;